samples/client/petstore/python-asyncio/tests/test_api_client.py

```diff
@@ -0,0 +1,31 @@
+# coding: utf-8
+
+# flake8: noqa
+
+import os
+import unittest
+import asyncio
+import weakref
+
+from tests.util import async_test
+import petstore_api
+
+HOST = 'http://localhost:80/v2'
```

Comment:
Can you delete HOST, I don't think we use it here

Reply:
Sure, removed.